Makefile

```diff
@@ -377,10 +387,14 @@ cns-image-name-and-tag: # util target to print the CNS image name and tag.
 
 cns-image: ## build cns container image.
 	$(MAKE) container \
```

Comment:
should we define a BUILDX_ACTION default value in the Makefile? Seems this target can be called directly and action in this case would be undefined. Not sure if this would break container-docker above

Reply:
fixed